### DISCUSSION TITLE
Bigger map sizes

### DISCUSSION THREAD

**Discussion by okpyl** [Keriew/augustus#865] [Ideas]
Hello,

As our modern computers can run the game pretty well could it be possible to have much bigger maps than the actual maximum (enormous)?

Indeed, it could totally change the game and help build giant cities.

Is it possible ?

**Comment by crudelios** [comment]
Hi,

It is and in fact is planned.

However the codebase is pretty rigid so many changes need to be made so the game's engine can support different city sizes, so it'll take a while.

**Reply by okpyl** [reply]
Thanks for your anwser! Looking forward to it!

Could you please share a possible timline? (or the link to the planned features that you will add next)?

Once again thanks for your hard work!

**Comment by crudelios** [comment]
Well, our current quem has been on adding new resources and improving custom scenarios.

We don't have a fixed timberline for the larger maps but considering the lack of developers and the fact that I currently have little time for the project as well, it's likely still a few years before it gets implemented.
